### PR TITLE
feat(js-cypress): readme update

### PR DESCRIPTION
**Pull request by nguyeda1**
- updated README
- moved main.js to src/main.js
- added code comments

closes apify/apify-web#2673

### CODE REVIEW

**Review by HonzaTuron — APPROVED** [review]
Very well commented code 👍

**Comment by nguyeda1** [comment]
> Very well commented code 👍

Inspired by @jbartadev :)

File: templates/js-cypress/cypress.config.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
 import { defineConfig } from 'cypress';
 
 export default defineConfig({
-    e2e: {
-        // to be able to run the tests locally
-        baseUrl: 'https://apify.com',
-    },
+    e2e: {},
```
Comment (jbartadev) [inline templates/js-cypress/cypress.config.js:4]: Why did you remove the `baseUrl` ?

File: templates/js-cypress/README.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 
-You can also customize the test run by specifying other options in the settings, e.g. the screen size, video recording or the default command timeout.
+## How it works
 
-After running the tests, the Apify Platform stores the results in a comprehensive way - datasets for json results, key value store for videos. You can view the results directly on the platform or download them to your local machine using a REST API.
+You can easily run your tests on Apify Platform, just copy-paste your test files into `cypress/e2e` folder. The tests' names need to end with `-spec.cy.js`.
+
+You can also customize the test run by specifying other options in the input, e.g. the screen size, video recording or the default command timeout.
 
+After running the tests, the Apify Platform stores the results in a comprehensive way - datasets for json results, key value store for videos. You can view the results directly on the platform or download them to your local machine using a REST API.
```
Comment (jbartadev) [inline templates/js-cypress/README.md:18]:
```suggestion
You can easily run your tests on the Apify Platform, just copy-paste your test files into `cypress/e2e` folder. The tests' names need to end with `-spec.cy.js`.

You can also customize the test run by specifying other options in the input, e.g. the screen size, video recording, or the default command timeout.

After running the tests, the Apify Platform stores the results in a comprehensive way - datasets for JSON results, and key-value store for videos. You can view the results directly on the platform or download them to your local machine using a REST API.
```

**Review by jbartadev — APPROVED** [review]
Nice. Preapproving.

File: templates/js-cypress/cypress.config.js (hunk 1 above)
Comment (nguyeda1) [reply templates/js-cypress/cypress.config.js:4]: Because the CLI update should cover this and I did not want to confuse users with default values on too many places.. But now that I think about it might not work with older CLI version so I will add it back
